stable/drone/templates/deployment-agent.yaml

```diff
@@ -54,6 +54,14 @@ spec:
           - name: {{ $key }}
             value: {{ $value | quote }}
           {{- end }}
+        livenessProbe:
+          httpGet:
+            path: "/healthz"
+            port: 3000
```

Comment:
can you please use the service port name instead of the value?

Reply:
can you, please,  also add those settings as well:
- `initialDelaySeconds`
- `periodSeconds`
- `timeoutSeconds`
- `successThreshold`
- `failureThreshold`

Reply:
@cpanato As tweakable options? The defaults work as is.

Reply:
yes. what do you think?

Reply:
@cpanato Just feels like options for the sake of having options to me. `deployment-server.yaml` already has a livenessProbe without having these options.

Reply:
ok so ignore my comment